### PR TITLE
Javascript Project: Leaderboard - final touches

### PR DESCRIPTION
**Pull request by yigitm**
Below updates are done;

- The page style is completed according to the wireframe design.
- Readme file screenshot is changed
- Overlap issues are fixed & a media query is also added.

Thank you for your time.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,24 +7,26 @@
...
-          <h2>Add your score</h2>
-          <input id="name" type="text" placeholder="Your name" />
-          <input id="score" type="text" placeholder="Your score" />
-          <button type="submit">Submit</button>
-        </div>
-      </section>
-    </main>
-    <footer>@microverse-inc</footer>
+    <div class="grid-container">
+      <header><h1>Leaderboard</h1></header>
```
Comment (GzimAsani) [inline src/index.html:11]: - [x] Please spend a little bit more time to fix this overlapping between the scores and your footer, it'll just look better
Screenshot from 2021-12-16 20-53-02

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @yigitm                                    
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However                                 
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project it looks really good :muscle: 

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (yigitm) [reply src/index.html:11]: Overlap issues are fixed & a media query is also added.

Thank you for the review.

**Review by leolpaz — APPROVED** [review]
Hi @yigitm , 

Your project is complete!
Congratulations! 🎉🎉 

To highlight:

- Congratulations on implementing the changes requested by the previous reviewer

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**